jit-application/src/main/resources/application.yml

```diff
@@ -23,9 +44,30 @@ spring:
   profiles:
     active: nosecurity
     group:
-      dev: nosecurity, logsql, localdb
-      test: nosecurity, logsql, localdb
+      dev: nosecurity, logsql, localdb, fakesmtpserver
+      test: nosecurity, logsql, localdb, fakesmtpserver
       prod: default
+  mail:
+    # Syntactically correct host name that will fail to resolve
+    # in DNS.  The hostname should be guide most admins to the
+    # proper solution. :)
+    host: please-set-spring.mail.host-property-in-dsca-app.invalid
```

Comment:
Typo in `dcsa` - probably copy-wasted from me :)

Reply:
Fixed